src/screens/WhotoFollow/index.js

```diff
@@ -199,7 +176,7 @@ const WhotoFollow = () => {
           try {
             const userId = await JwtDecode(res.token).user_id;
             const anonymousUserId = await JwtDecode(res.anonymousToken).user_id;
-            setProfileId({
+            setAuth({
               anonProfileId: anonymousUserId,
               signedProfileId: userId,
               token: res.token,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [179-183]

The change from `setProfileId` to `setAuth` suggests that the developer has refactored the user authentication state management. This could be a positive change if the new approach provides better encapsulation or improved functionality.
```diff
-            setProfileId({
+            setAuth({
               anonProfileId: anonymousUserId,
               signedProfileId: userId,
               token: res.token,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->